src/app/map/Tile/Empty/empty.tsx

```diff
@@ -62,17 +66,31 @@ export function DynamicEmptyTile(props: DynamicEmptyTileProps) {
   const coord = CoordSystem.parseId(props.coordId);
   const userOwnsThisSpace =
     props.currentUserId !== undefined && coord.userId === props.currentUserId;
+  
+  // Prepare drop handlers if this is a valid drop target
+  const dropProps = isValidDropTarget && tileActions ? {
```

Comment:
drogProps should be initialized with a helper function below this DynamicEmptyTile component. Overall this dropProps should be the only line in the component (with the one in the render using dropProps). This way if someone is dealing with an issue with drag and drop he knows where to look and if he does not care it limits the cognitive load.